packages/kbn-pm/src/utils/projects.test.ts

```diff
@@ -31,8 +32,32 @@ import {
 } from './projects';
 
 const rootPath = resolve(`${__dirname}/__fixtures__/kibana`);
+const symlinkSourcePath = join(__dirname, '__fixtures__/symlinked-plugins/corge');
+const symlinkDestinationPath = join(rootPath, 'plugins/corge');
+
+const handleError = (accept: () => void, reject: (err: Error) => void) => (err: Error) => {
```

Comment:
can we use `import { promisify } from 'util';` here?

Reply:
Should be able to, good catch!